Jenkinsfile

```diff
@@ -0,0 +1,14 @@
+pipeline {
+    agent any
+
+    stages {
+        stage('Java Program') {
+            steps {
+                bat ''' javac DuplicateString.java
```

Comment:
there appears to be a typo here, the file is called [DuplicatesString.java](https://github.com/infuse-training-jan-2023/Practice-Jeston/pull/9/files#diff-1019d7e12fc7e3a1f6ca39970b5248a03e13941e17ca0866237be871ab2fed47) @jeston13